hw24di/src/main/java/ru/otus/homework/controller/AdminController.java

```diff
@@ -0,0 +1,54 @@
+package ru.otus.homework.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import ru.otus.homework.api.service.DbServiceException;
+import ru.otus.homework.model.User;
+import ru.otus.homework.service.UserServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+public class AdminController{
+    private final String TEMPLATE_VARIABLE_USERS = "users";
+    private final String TEMPLATE_VARIABLE_STATUS = "status";
+    private final String ADMIN_FILE_NAME = "Admin";
+    private final String USER_CREATED_TEXT = "User is created";
+    private final String USER_EXISTS_TEXT = "User is already exist";
+
+    private final UserServiceImpl userService;
```

Comment:
Здесь должен быть интерфейс
```suggestion
    private final UserService userService;
```

Reply:
Принято